### PR TITLE
[14.0][ADD] partner_title_order

### PR DESCRIPTION
**Pull request by kv1612**

### CODE REVIEW

File: partner_title_order/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* Simone Orsi <simahawk@users.noreply.github.com>
```
Comment (simahawk) [inline partner_title_order/readme/CONTRIBUTORS.rst]:
```suggestion
* Simone Orsi <simone.orsi@camptocamp.com>
```

**Review by simahawk — CHANGES_REQUESTED** [review]
pre-commit complains

File: partner_title_order/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "version": "14.0.1.0.0",
+    "category": "Hidden",
+    "author": "Camptocamp",
+    "license": "AGPL-3",
+    "depends": [
+        "base",
+    ],
+    "website": "https://github.com/OCA/partner-contact",
+    "data": [
+        # views
```
Comment (TDu) [inline partner_title_order/__manifest__.py]:
```suggestion
```
Unhelpful comment

File: partner_title_order/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,5 @@
+===================
+Partner title order
+===================
```
Comment (TDu) [inline partner_title_order/readme/DESCRIPTION.rst]: Those 3 first lines should be removed

File: partner_title_order/__manifest__.py (hunk 2 above)
Comment (kv1612) [reply partner_title_order/__manifest__.py]: I have just updated. Thank you !

File: partner_title_order/readme/DESCRIPTION.rst (hunk 3 above)
Comment (kv1612) [reply partner_title_order/readme/DESCRIPTION.rst]: Done

**Comment by kv1612** [comment]
Thanks @TDu, @simahawk for your review. This PR has been updated.

**Review by TDu — APPROVED** [review]

**Review by sebalix — APPROVED** [review]

**Comment by sebalix** [comment]
@kv1612 pre-commit still failing, can you re-run `pre-commit run -a`?

**Review by simahawk — APPROVED** [review]

**Review by simahawk — CHANGES_REQUESTED** [review]
keeping it blocked till pre-commit is happy

**Comment by kv1612** [comment]
Hi @simahawk, @sebalix ! Sorry, I forgot run pre-commit. This PR has been updated. Thank you !

File: partner_title_order/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (NL66278) [inline partner_title_order/__manifest__.py]: Always use `https:` for the license lines (and where possible for all other links).

**Review by NL66278 — APPROVED** [review]
:+1: Except for license lines remark, LGTM.

File: partner_title_order/__manifest__.py (hunk 4 above)
Comment (simahawk) [reply partner_title_order/__manifest__.py]: @kv1612 can you update all links? :pray:

Comment (kv1612) [reply partner_title_order/__manifest__.py]: Hi @simahawk, this PR has been updated !!!

**Review by simahawk — APPROVED** [review]

**Pull request merged**